client/src/components/_common/InfiniteScrollTable/InfiniteScrollTable.scss

```diff
@@ -36,42 +36,34 @@
   }
 
   tbody {
-    tr:not([class=-status]) td {
+    tr:not(.-status) td {
       border-bottom: 1px solid rgba(0, 0, 0, 0.1);
     }
-
-    tr:nth-child(even):not([class=-status]) {
+    tr:nth-child(even):not(.-status) {
       background-color: rgba(0, 0, 0, 0.03);
     }
-    tr:hover:not([class=-status]) {
+    tr:hover:not(.-status) {
       background-color: rgba(0, 0, 0, 0.05);
     }
```

Comment:
Yes, `[class=-status]` === `.-status`. The `[class=-status]` is either an attempt to write, or an update after having written, `[class*=-status]` (which would select `.-status` and `.status-any-suffix-ever`.

_P.S. if you are interested in the removed syntax, then know that it would be more clearly written as `[class="-status"]` and `[class*="-status"]`._